test/test_core.py

```diff
@@ -3758,6 +3758,20 @@ def test_dlfcn_asyncify(self):
       '''
     self.do_run(src, 'before sleep\nafter sleep\n42\n')
 
+  @requires_jspi
+  @needs_dylink
+  def test_dlfcn_jspi(self):
+    side_flags = ['-sSIDE_MODULE']
+    if self.get_setting('MEMORY64'):
+      side_flags.append('-sMEMORY64')
+    self.set_setting('JSPI')
```

Comment:
`requires_jspi` already takes care of setting this so I think this line can be removed.